apps/studio/components/layouts/StorageLayout/StorageBucketsLayout.tsx

```diff
@@ -31,10 +39,16 @@ export const StorageBucketsLayout = ({ children }: PropsWithChildren) => {
         ]
       : []
 
+  useEffect(() => {
+    if (!isStorageV2) router.replace(`/project/${ref}/storage/buckets`)
+  }, [isStorageV2, ref, router])
+
   return (
     <PageLayout
-      title={config?.displayName || 'Storage'}
-      subtitle={config?.description || 'Manage your storage buckets and files.'}
+      title={title || (config?.displayName ?? 'Storage')}
+      subtitle={
+        hideSubtitle ? config?.description || 'Manage your storage buckets and files.' : null
```

Comment:
FYI @joshenlim I’ve inverted this ternary in [the upcoming PR](https://github.com/supabase/supabase/pull/39342/files#diff-b6ba82aef3e975e7844954dc1181988785d1517ad63aeba4295c77025920e6beL49) because this hides the subtitle text. Let me know if that was intentional (I can see merit for leaving it off!)